packages/addresses/src/addresses.helpers.ts

```diff
@@ -1,9 +1,8 @@
-import dotenv from "dotenv";
-import { readdirSync, readFileSync, writeFileSync } from "fs";
 import { network } from "hardhat";
```

Comment:
Can we rename the `async` methods to something more explicit? e.g.: `readLocalnetListFromFile`.

And rename `loadAddressFromFile` to `getAddress` since that will be the main method we expose via NPM package and it's better to keep it with a clean name and not exposing internals.